src/main/java/net/minecraftforge/client/event/RenderTooltipEvent.java

```diff
@@ -259,30 +375,61 @@ public PostText(@Nonnull ItemStack stack, @Nonnull List<? extends FormattedText>
         private final int originalBackground;
         private final int originalBorderStart;
         private final int originalBorderEnd;
-        private int background;
+        private int backgroundStart;
+        private int backgroundEnd;
         private int borderStart;
         private int borderEnd;
 
-        public Color(@Nonnull ItemStack stack, @Nonnull List<? extends FormattedText> textLines, PoseStack matrixStack, int x, int y, @Nonnull Font fr, int background, int borderStart,
-                int borderEnd)
+        public Color(@Nonnull ItemStack stack, @Nonnull List<? extends FormattedText> textLines, PoseStack matrixStack, int x, int y, @Nonnull Font fr, int background, int borderStart, int borderEnd)
         {
-            super(stack, textLines, matrixStack, x, y, fr);
+            this(stack, matrixStack, x, y, fr, background, borderStart, borderEnd, List.of());
```

Comment:
While it is a bit more of a mess, I feel like this should more or less be a copy of the new Color constructor instead of calling this, so that it can then pass `lines` to the now deprecated `RenderTooltipEvent` super constructor, in case any mods are currently listening to the events and any other mods are still using/decide to use the GuiUtils methods for some reason.